data/brands/shop/cosmetics.json

```diff
@@ -577,6 +578,34 @@
         "shop": "cosmetics"
       }
     },
+    {
+      "displayName": "Рив Гош",
+      "id": "Rive-Gauche",
```

Comment:
you don't need to add id manually, it will be generated. And what you've written, the word "Rive-Gauche", will be overwrite by program.